apps/camera_pipe/process.cpp

```diff
@@ -1,10 +1,37 @@
-#include <arpa/inet.h>
 #include <stdint.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <assert.h>
+
```

Comment:
There's another PR out that will simplify the timing here much more using C++11 chrono stuff. I'd revert the changes in this file.